app/src/main/java/com/example/synerzip/recircle_android/ui/ListedItemActivity.java

```diff
@@ -53,7 +58,7 @@ public void btnViewItem(View view) {
      */
```

Comment:
handle back press in this activity, after back press it should go to the home activity